dotcom-rendering/.eslintrc.js

```diff
@@ -1,5 +1,34 @@
 const transitionRules = require('./eslint-guardian');
 
+/** TODO: Review these */
+const rulesToReview = {
+	'import/no-extraneous-dependencies': 'warn', // 683 problems
+	'consistent-return': 'warn', // 51 problems
+	'default-case': 'warn', // 50 problems
+	'react/no-danger': 'warn', // 48 problems
+	'no-underscore-dangle': 'warn', // 45 problems
+	'react/no-array-index-key': 'warn', // 34 problems
+	'react/button-has-type': 'warn', // 23 problems
+	'@typescript-eslint/require-await': 'warn', // 22 problems
+	'react/jsx-curly-newline': 'warn', // 8 problems
+	'no-case-declarations': 'warn', // 7 problems
+	'no-useless-escape': 'warn', // 6 problems
+};
```

Comment:
This is very cool

Reply:
https://github.com/guardian/dotcom-rendering/blob/mxdvl/eslint/count-offending-rules/scripts/deno/eslint.ts